JsonPreprocessor/CJsonPreprocessor.py

```diff
@@ -501,7 +502,16 @@ def jsonLoad(self, jFile : str, masterFile : bool = True):
         except Exception as reason:
             raise Exception(f"Could not read json file '{jFile}' due to: '{reason}'!")
 
-        
+        for line in sJsonData.splitlines():
```

Comment:
Hi Son,
I have a concern here.
splitlines() will most likely not work if somebody makes:
{ "key" : "abc
              def" }
Can you please check if this is critial. I'm not 100% sure that json-syntax allows this. If it is allowed, then we have a problem here.

Thank you,
Thomas

Reply:
Hello Thomas,

    I have just checked, json file dose not allow the syntax that you are mentioned.

Thank you,
Son